#!/usr/bin/env python3
"""
MovementData        Define movement for robot      
"""


class MovementData:
    
    def __init__(self):
        self.serialID = 0
        self.driveDirection = 0
        self.stop = 0
        self.packIn = 0
        self.packOut = 0
        self.pause = 0
        self.msg = ""
        return


